openmdao/recorders/__init__.py

```diff
@@ -0,0 +1,6 @@
+# <<<<<<< HEAD
+# from openmdao.recorders.base_recorder import BaseRecorder
```

Comment:
Sorry. Fixed…

On Oct 19, 2015, at 3:11 PM, Kenneth Moore <notifications@github.com<mailto:notifications@github.com>>
 wrote:


In openmdao/recorders/__init__.py<https://github.com/OpenMDAO/OpenMDAO/pull/357#discussion_r42413109>:

> @@ -0,0 +1,6 @@
> +# <<<<<<< HEAD
> +# from openmdao.recorders.base_recorder import BaseRecorder


merge conflict? I guess it is commented out, but maybe we should clean it all out

—
Reply to this email directly or view it on GitHub<https://github.com/OpenMDAO/OpenMDAO/pull/357/files#r42413109>.

---------
Herb Schilling
Scientific Applications and Visualization Team
http://ocio.grc.nasa.gov/gvis
Office of the Chief Information Officer
NASA Glenn Research Center
hschilling@nasa.gov<mailto:hschilling@nasa.gov>